roles/openshift_descheduler/defaults/main.yaml

```diff
@@ -48,3 +48,4 @@ openshift_descheduler_cronjob_schedule: "*/1 0 * * *"
 # descheduler container setup
 openshift_descheduler_dry_run_mode: true
 openshift_descheduler_node_selectors: null
+openshift_descheduler_debug_level: 2
```

Comment:
Maybe `"openshift_descheduler_debug_level: {{ debug_level | default(2) }}"` ? As is at https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_control_plane/defaults/main.yml#L5.

Reply:
Since descheduler has only a few debug levels as compared to master, it wont work exactly same way.

Reply:
@michaelgugino @sdodson @mtnbikenc  any opinion on that?

Reply:
I can not tie descheduler debug_level to cluster debug level as descheduler has its own debug level not tied to cluster debug level.

Reply:
> Since descheduler has only a few debug levels as compared to master, it wont work exactly same way.

Why can't it work the same way as other components? If it only supports 1,2,3 but I set 10000 then it should just behave as if i'd set 3, right?

Reply:
There's a lot of value in having all the components behave the same was as each other.

Reply:
> Why can't it work the same way as other components? If it only supports 1,2,3 but I set 10000 then it should just behave as if i'd set 3, right?

But 4 might not be a too much level for cluster debugs, but can be too much for descheduler.

Reply:
My opinion is to align all components on debug level where possible.

If that's not possible, then whatever seems most appropriate for the component is fine with me.

Reply:
@aveshagarwal They can still set `openshift_descheduler_debug_level=4` and leave other components at the global default if they'd like.